controllers/nutanixcluster_controller.go

```diff
@@ -250,8 +263,7 @@ func (r *NutanixClusterReconciler) reconcileNormal(rctx *nctx.ClusterContext) (r
 
 	err := r.reconcileCategories(rctx)
 	if err != nil {
-		errorMsg := fmt.Errorf("failed to reconcile categories for cluster %s: %v", rctx.Cluster.Name, err)
-		rctx.SetFailureStatus(capierrors.CreateClusterError, errorMsg)
+		// Don't return fatal error but keep retrying until categories are created.
```

Comment:
we need to log the error somewhere

Reply:
@deepakm-ntnx you will have it in the reconcile result log below no ???

Reply:
Added explicit log.